web-cheatsheet.adoc

```diff
@@ -229,12 +228,11 @@ include::includes-examples.adoc[tag=lists-other-types]
 
 +++</td></tr></table>+++
 
-
 == Links and anchors
 
 === Anchors
 
-*Block or a section* heading: in the attribute list include `[id=myid]` or the shorthand form `+[#myid]+` (as first).
```

Comment:
We should not promote using the id attribute directly. It's not wrong, but we don't want users getting the impression that is the way to do it.

Reply:
You mean to remove `[id=myid]` and only leave `+[#myid]+` ? Ok by me :)

Reply:
Correct.